spec/view_models/journey_navigation_spec.rb

```diff
@@ -0,0 +1,81 @@
+RSpec.describe JourneyNavigation do
+  subject(:journey_navigation) { JourneyNavigation.new(tree, current_step) }
+
+  let(:current_step) { 'other' }
+
+  let(:foo_guide) { instance_double(Guide, content_type: :govspeak, id: double, slug: 'foo', label: 'Foo') }
+  let(:bar_guide) { instance_double(Guide, content_type: :govspeak, id: double, slug: 'bar', label: 'Bar') }
+  let(:baz_guide) { instance_double(Guide, content_type: :govspeak, id: double, slug: 'baz', label: 'Baz') }
+
+  let(:tree) do
+    Tree::TreeNode.from_hash(
+      ['foo', foo_guide] => { ['bar', bar_guide] => { ['baz', baz_guide] => nil } }
+    )
+  end
+
+  describe '.overview' do
+    subject { journey_navigation.overview }
```

Comment:
I think the readability might be improved with a named subject, perhaps `overview_step`, or even just `step`?